[EDU-3190] feat: add test zone (#557)
aziontech/docs: src/content/docs/en/pages/secure-journey/intelligent-dns/test-zone.mdx

---
title: How to test an Intelligent DNS zone processing
description: You can test your zones to check if they're properly processing information.
meta_tags: secure, intelligent dns, zone, test, dig, host, nslookup
namespace: docs_guides_secure_test_zone
permalink: /documentation/products/guides/secure/test-zone/
menu_namespace: secureMenu
---

import Button from '~/components/Button.astro'

Once you've set up your [Intelligent DNS zone](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/), you can test it to see if it's properly processing your information.

See some of the available tools next.

> 
> ## DIG

Reviewer: ```suggestion

---

## DIG
```

> **Domain Information Groper (DIG)** is a network administration command-line tool. It's useful for network troubleshooting and for educational purposes. When a specific name server isn't specified in the command invocation, it uses the operating system's default resolver, usually configured in the file `resolv.conf`. Without any arguments, it queries the DNS root zone.

Reviewer: name server or nameserver?

> **Domain Information Groper (DIG)** is a network administration command-line tool. It's useful for network troubleshooting and for educational purposes. When a specific name server isn't specified in the command invocation, it uses the operating system's default resolver, usually configured in the file `resolv.conf`. Without any arguments, it queries the DNS root zone.

Author: ```suggestion
**Domain Information Groper (DIG)** is a network administration command-line tool. It's useful for network troubleshooting and for educational purposes. When a specific nameserver isn't specified in the command invocation, it uses the operating system's default resolver, usually configured in the file `resolv.conf`. Without any arguments, it queries the DNS root zone.
```

To test **Intelligent DNS**, you must use one of the [Azion nameservers](/en/documentation/products/intelligent-dns/#dns-provider-configuration) that you've configured in your DNS registry.

<Button href="/en/documentation/products/guides/run-the-dig-command/" text="go to dig command guide" variant="secondary"></Button>

> 
> ## HOST

Reviewer: ```suggestion

---

## HOST
```

**Host** queries the DNS, converting domain names into IP addresses and vice-versa. If no argument or option is provided, the host prints a brief summary of the command line options and arguments.

For example, if you have the following settings:

| Name | Type | Value                                               | TTL  |
|------|------|-----------------------------------------------------|------|
| www  | A    | 192.168.0.1<br /><br /> 192.168.0.2<br /><br /> 192.168.0.3 | 3600 |

To check how Intelligent DNS handles a type "A" query, registered to the “www” record, type:

```bash
host www.example.com ns1.aziondns.net
```

The response to this will be:

```bash
Using domain server: Name: ns1.aziondns.net Address: 179.191.160.2#53 Aliases: www.example.com has address 192.168.0.1 www.example.com has address 192.168.0.2 www.example.com has address 192.168.0.3
```

> 
> ## NSLOOKUP

Reviewer: ```suggestion

---

## NSLOOKUP
```

---
title: How to test an Intelligent DNS zone processing
description: You can test your zones to check if they're properly processing information.
meta_tags: secure, intelligent dns, zone, test, dig, host, nslookup
namespace: docs_guides_secure_test_zone
permalink: /documentation/products/guides/secure/test-zone/
menu_namespace: secureMenu
---

import Button from '~/components/Button.astro'

Once you've set up your [Intelligent DNS zone](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/), you can test it to see if it's properly processing your information.

> See some of the available tools next.

Reviewer 2: ```suggestion
```

---

## DIG

**Domain Information Groper (DIG)** is a network administration command-line tool. It's useful for network troubleshooting and for educational purposes. When a specific nameserver isn't specified in the command invocation, it uses the operating system's default resolver, usually configured in the file `resolv.conf`. Without any arguments, it queries the DNS root zone.

To test **Intelligent DNS**, you must use one of the [Azion nameservers](/en/documentation/products/intelligent-dns/#dns-provider-configuration) that you've configured in your DNS registry.

<Button href="/en/documentation/products/guides/run-the-dig-command/" text="go to dig command guide" variant="secondary"></Button>

---

> ## HOST

Reviewer 2: ```suggestion
## Test using HOST command
```

**Host** queries the DNS, converting domain names into IP addresses and vice-versa. If no argument or option is provided, the host prints a brief summary of the command line options and arguments.

For example, if you have the following settings:

| Name | Type | Value                                               | TTL  |
|------|------|-----------------------------------------------------|------|
| www  | A    | 192.168.0.1<br /><br /> 192.168.0.2<br /><br /> 192.168.0.3 | 3600 |

> To check how Intelligent DNS handles a type "A" query, registered to the “www” record, type:

Reviewer 2: ```suggestion
To check how Intelligent DNS handles a type A query, registered to the `www` record, run:
```

```bash
host www.example.com ns1.aziondns.net
```

The response to this will be:

```bash
Using domain server: Name: ns1.aziondns.net Address: 179.191.160.2#53 Aliases: www.example.com has address 192.168.0.1 www.example.com has address 192.168.0.2 www.example.com has address 192.168.0.3
```

---

## NSLOOKUP

**nslookup** is a tool used to obtain information about DNS records for a particular domain, host, or IP.

For example, if you have the following settings:

| Name | Type | Value                                               | TTL  |
|------|------|-----------------------------------------------------|------|
| www  | A    | 192.168.0.1<br /><br /> 192.168.0.2<br /><br /> 192.168.0.3 | 3600 |

> To check how Intelligent DNS handles a type "A" query, registered to the “www” record, type the following command:

Reviewer 2: ```suggestion
To check how Intelligent DNS handles a type A query, registered to the `www` record, run the following command:
```
